packages/cli/test/integration/shared/testDb.ts

```diff
@@ -681,7 +681,7 @@ export const getMySqlOptions = ({ name }: { name: string }): ConnectionOptions =
 async function encryptCredentialData(credential: CredentialsEntity) {
 	const encryptionKey = await UserSettings.getEncryptionKey();
 
-	const coreCredential = createCredentiasFromCredentialsEntity(credential, true);
+	const coreCredential = createCredentialsFromCredentialsEntity(credential, true);
```

Comment:
If what we get is a core credential, then maybe `createCoreCredentialFromEntity()` or just `createCoreCredential`?